Source/scrollrt.cpp

```diff
@@ -893,7 +893,8 @@ void DrawView(int StartX, int StartY)
 	} else if (questlog) {
 		DrawQuestLog();
 	}
-	if (!chrflag && plr[myplr]._pStatPts != 0 && !spselflag) {
+	if (!chrflag && plr[myplr]._pStatPts != 0 && !spselflag
+		&& (!questlog || SCREEN_HEIGHT >= SPANEL_HEIGHT + PANEL_HEIGHT + 49 || SCREEN_WIDTH >= 4 * 320)) {

```

Comment:
what's this

Reply:
why 4* 320 instead of 1280? :D

Reply:
Space for 4 * panels. 1280 is a magic number with no meaning :P

Reply:
`4 * PANEL_WIDTH` would make more sense then 🙂

Reply:
sadly PANEL_WIDTH is 640 as it's the main panel's width - there's no define for 320 - side panel width :D

Reply:
It might make sens to add `SPANEL_WIDTH 320`